test/ghtype_tests.jl

```diff
@@ -421,6 +421,43 @@ end
     test_show(pr_result)
 end
 
+@testset "PullRequestFile" begin
```

Comment:
Better to have the test data point to something in this repo itself, instead of depending on a different users repo?

Reply:
I won't matter since these test don't make any API calls to GitHub. Its just for testing the JSON to struct conversion and the `name` method.

Reply:
okay